py4cast/lightning.py

```diff
@@ -301,22 +370,22 @@ def inspect_tensors(self):
     @rank_zero_only
     def log_hparams_tb(self):
         if self.logging_enabled and self.logger:
-            hparams = self.hparams["hparams"]
+            hparams = self.hparams
             # Log hparams in tensorboard hparams window
-            dict_log = asdict(hparams)
+            dict_log = hparams
             dict_log["username"] = getpass.getuser()
             self.logger.log_hyperparams(dict_log, metrics={"val_mean_loss": 0.0})
             # Save model & dataset conf as files
-            if hparams.dataset_conf is not None:
+            if self.hparams["dataset_conf"] is not None:
                 shutil.copyfile(
-                    hparams.dataset_conf, hparams.save_path / "dataset_conf.json"
+                    self.hparams["dataset_conf"], self.hparams.save_path / "dataset_conf.json"
                 )
-            if hparams.model_conf is not None:
+            if hparams["model_conf"] is not None:
                 shutil.copyfile(
-                    hparams.model_conf, hparams.save_path / "model_conf.json"
+                    hparams["model_conf"], self.hparams.save_path / "model_conf.json"
```

Comment:
I'm surprised that you use hparams like a dict (`self.hparams["truc"]`), should't it be `self.hparams.truc` ?